examples/flutter_gallery/lib/demo/progress_indicator_demo.dart

```diff
@@ -73,6 +74,7 @@ class _ProgressIndicatorDemoState extends State<ProgressIndicatorDemo> with Sing
       new Row(
         mainAxisAlignment: MainAxisAlignment.spaceEvenly,
         children: <Widget>[
+          new CupertinoActivityIndicator(),
```

Comment:
We can talk about this more in person if you like, but for now, I think it will be easier if we put the cupertino widgets in their own section on the home screen and we don't mix cupertino and non-cupertino in the same demo.